packages/RNTester/js/examples/Alert/AlertExample.js

```diff
@@ -9,137 +9,169 @@
-class SimpleAlertExampleBlock extends React.Component<Props> {
-  render() {
-    return (
-      <View>
-        <TouchableHighlight
-          style={styles.wrapper}
-          onPress={() => Alert.alert('Alert Title', alertMessage)}>
-          <View style={styles.button}>
-            <Text>Alert with message and default button</Text>
-          </View>
-        </TouchableHighlight>
-        <TouchableHighlight
-          style={styles.wrapper}
-          onPress={() =>
-            Alert.alert('Alert Title', alertMessage, [
-              {text: 'OK', onPress: () => console.log('OK Pressed!')},
-            ])
-          }>
-          <View style={styles.button}>
-            <Text>Alert with one button</Text>
-          </View>
-        </TouchableHighlight>
-        <TouchableHighlight
-          style={styles.wrapper}
-          onPress={() =>
-            Alert.alert('Alert Title', alertMessage, [
-              {text: 'Cancel', onPress: () => console.log('Cancel Pressed!')},
-              {text: 'OK', onPress: () => console.log('OK Pressed!')},
-            ])
-          }>
-          <View style={styles.button}>
-            <Text>Alert with two buttons</Text>
-          </View>
-        </TouchableHighlight>
-        <TouchableHighlight
-          style={styles.wrapper}
-          onPress={() =>
-            Alert.alert('Alert Title', null, [
-              {text: 'Foo', onPress: () => console.log('Foo Pressed!')},
-              {text: 'Bar', onPress: () => console.log('Bar Pressed!')},
-              {text: 'Baz', onPress: () => console.log('Baz Pressed!')},
-            ])
-          }>
-          <View style={styles.button}>
-            <Text>Alert with three buttons</Text>
-          </View>
-        </TouchableHighlight>
-        <TouchableHighlight
-          style={styles.wrapper}
-          onPress={() =>
-            Alert.alert(
-              'Foo Title',
-              alertMessage,
-              '..............'.split('').map((dot, index) => ({
-                text: 'Button ' + index,
-                onPress: () => console.log('Pressed ' + index),
-              })),
-            )
-          }>
-          <View style={styles.button}>
-            <Text>Alert with too many buttons</Text>
-          </View>
-        </TouchableHighlight>
-        <TouchableHighlight
-          style={styles.wrapper}
-          onPress={() =>
-            Alert.alert(
-              'Alert Title',
-              null,
-              [{text: 'OK', onPress: () => console.log('OK Pressed!')}],
-              {
-                cancelable: false,
+
+const AlertWithDefaultButton = () => {
+  const alertMessage = 'An external USB drive has been detected!';
+
+  return (
+    <View>
+      <TouchableHighlight
+        testID="alert-with-default-button"
+        style={styles.wrapper}
+        onPress={() => Alert.alert('Alert', alertMessage)}>
+        <View style={styles.button}>
+          <Text>Tap to view alert</Text>
+        </View>
```

Comment:
should we use a button component instead?

Reply:
I prefer using the Touchables because buttons have more contrast and all the focus would go on them. However, the focus should be on the title and description.